src/args.rs

```diff
@@ -51,7 +51,7 @@ pub struct Args {
     pub list_all: bool,
 
     /// Autoconfigure
-    #[arg(short, long, value_name = "classid", conflicts_with_all(["install", "remove"]))]
+    #[arg(short, long, value_name = "classid", conflicts_with_all(["install", "remove"]), default_missing_value = "any", num_args(0..=1), require_equals = true)]
```

Comment:
I think equals should not be forced here

Reply:
Unfortunately, clap requires to specify `require_equals` to use `default_missing_value`

https://docs.rs/clap/latest/clap/struct.Arg.html#method.default_missing_value

Reply:
Hmm, it looks like `require_equals = true` can indeed be removed, so documentation is misleading with its note.